tests/podman/Dockerfile

```diff
@@ -3,18 +3,17 @@ FROM fedora:latest
 ENV GOPATH=/root/go
 ENV PATH=/usr/bin:/usr/sbin:/root/go/bin:/usr/local/bin::/usr/local/sbin
 
-RUN yum install -y golang python git gcc automake autoconf libcap-devel \
+RUN dnf install -y golang python git gcc automake autoconf libcap-devel \
     systemd-devel yajl-devel libseccomp-devel go-md2man \
-    glibc-static python3-libmount libtool make podman xz nmap-ncat && \
-    dnf install -y 'dnf-command(builddep)' && dnf builddep -y podman && \
-    go get github.com/onsi/ginkgo/ginkgo && \
-    go get github.com/onsi/gomega/... && \
-    mkdir -p /root/go/src/github.com/containers && \
-    chmod 755 /root && \
-    (cd /root/go/src/github.com/containers && git clone -b main https://github.com/containers/podman && \
-     cd podman && \
-     make install.catatonit && \
-     make)
+    glibc-static python3-libmount libtool make podman xz nmap-ncat \
+    'dnf-command(builddep)' && \
+    dnf builddep -y podman && \
+    git clone -b main https://github.com/containers/podman /root/go/src/github.com/containers/podman && \
+    cd /root/go/src/github.com/containers/podman && \
```

Comment:
1. AFAIK there's no need to specify branch if it is a default one (for podman, `main` is indeed the default branch).
2. No need to use `go/src/<package_name>` directory any more (since Go 1.7 or so).

So you can simplify the above to

```
git clone https://github.com/containers/podman && \
cd podman && 
```

here.

Reply:
Quite right, updating it.

Reply:
Hmm, the podman test seems to have failed while looking for things in multiple locations, one of which is under $GOPATH, so maybe not.